docs/4.4/enterprise/ssh-rbac.md

```diff
@@ -263,11 +272,11 @@ spec:
       'environment': ['test', 'staging']
       # regular expressions are also supported, for example the equivalent
       # of the list example above can be expressed as:
-      'environment': '{% raw %}{{regexp.match("^test|staging$")}}{% endraw %}'
+      'environment': "{% raw %}{{regexp.match("^test|staging$")}}{% endraw %}"
```

Comment:
I don't think you can do this because the internal match condition uses `"` already.

Either:
```suggestion
      'environment': "{% raw %}{{regexp.match('^test|staging$')}}{% endraw %}"
```

Or:
```suggestion
      'environment': '{% raw %}{{regexp.match("^test|staging$")}}{% endraw %}'
```